mex/common/backend_api/connector.py

```diff
@@ -174,20 +245,25 @@ def preview_merged_item(
         )
         return PreviewModelTypeAdapter.validate_python(response)
 
-    def fetch_preview_items(
+    def fetch_preview_items(  # noqa: PLR0913
         self,
-        query_string: str | None,
-        entity_type: list[str] | None,
-        had_primary_source: list[str] | None,
-        skip: int,
-        limit: int,
+        *,
+        query_string: str | None = None,
+        identifier: str | None = None,
+        entity_type: list[str] | None = None,
+        referenced_identifier: list[str] | None = None,
+        reference_field: str | None = None,
+        skip: int = 0,
+        limit: int = 100,
     ) -> PaginatedItemsContainer[AnyPreviewModel]:
         """Fetch merged item previews that match the given set of filters.
 
         Args:
             query_string: Full-text search query
+            identifier: Optional merged item identifier filter
             entity_type: The items' entityType
-            had_primary_source: The items' hadPrimarySource
+            referenced_identifier: Optional merged item identifiers filter
+            reference_field: Optional field name to filter for
             skip: How many items to skip for pagination
             limit: How many items to return in one page
```

Comment:
All arguments are optional. Can we either add "optional" to all or remove it for all args and assume that devs infer that args with a default are optional.

Reply:
fixed dc9451d